webapp/app/store.js

```diff
@@ -38,13 +39,13 @@ const createReducer = (asyncReducers) =>
 // App middleware
 const middleware = [createDebounce(), thunkMiddleware, appErrorsMiddleware]
 
-if (ProcessUtils.isEnvDevelopment) {
-  const { logger } = require('redux-logger')
-  middleware.push(logger)
-}
-
 // App store
-export const store = createStore(createReducer({}), applyMiddleware(...middleware))
+export const store = createStore(
+    createReducer({}),
+    composeWithDevTools(
+      applyMiddleware(...middleware)
+    )
+)
```

Comment:
It is a good idea to check if the browser has the extension, to avoid getting undefined error:
`typeof window === 'object' && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__`
https://github.com/zalmoxisus/redux-devtools-extension/issues/320